configurations/operator/perf-serverless-thrpt-2vcpu.yaml

```diff
@@ -0,0 +1,17 @@
+email_subject_postfix: 'Scylla-Cloud backend'
+email_recipients: ['scylla-perf-results@scylladb.com', 'operator@scylladb.com']
+
+k8s_scylla_cluster_name: "sct-perf-thrpt-2vcpus"
+user_prefix: "perf-thrpt-2vcpus"
+
+n_loaders: 1
+instance_type_loader: 'c5.2xlarge'
+round_robin: true
+# https://github.com/scylladb/scylla-cluster-tests/issues/5883
+use_hdr_cs_histogram: false
+
+# NOTE: following are tmp values for testing 2vCPUs/16GbRAM single serverless Scylla cluster
+prepare_write_cmd: "cassandra-stress write no-warmup cl=ALL n=50331648 -schema 'replication(factor=3)' -mode cql3 native -rate threads=340 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..50331648"
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=30m -schema 'replication(factor=3)' -mode cql3 native -rate threads=340 -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..50331648,25165824,1006632)' "
+stress_cmd_r: "cassandra-stress read no-warmup cl=QUORUM duration=30m -schema 'replication(factor=3)' -mode cql3 native -rate threads=420 -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..50331648,25165824,1006632)' "
```

Comment:
After re-testing turned out that the optimal number of threads for read is 550 (and half of that for 1vcpu)

Reply:
@roydahan the results came from the same test, the 420 is for 15 minutes, 550 for 30 minutes.

Reply:
@rayakurl in our google doc the 550 threads is used for the `latency` test, not `throughput` which is here.

Reply:
@vponomaryov - we re-tested the read test for throughput as well, I'll make sure that the doc is updated